Stores/PageStore/Sources/PageStore/PageStore.swift

```diff
@@ -45,22 +46,38 @@ public final class PageStore: ObservableObject {
 // MARK: - Actions
 
 public extension PageStore {
+
+    // MARK: - Public methods
     
     func loadQuestions(_ type: LoadingType = .reload) {
```

Comment:
- [ ] Remove `type: LoadingType` at all because it is a rudiment of the previous implementation.

The next functions should be enough:
```swift
func loadNextQuestions() {
    guard dataManager.hasMoreData else { return }
    loadingMore = true
    dataManager.fetch { [unowned self] result in
        loadingMore = false
        switch result {
        case let .success(models):
            self.state = models.isEmpty ? .emptyContent : .content(models)
        case let .failure(error):
            self.state = .error(error)
        }
    }
}

func reloadQuestions() {
    dataManager.reset()
    state = .loading
    dataManager.fetch { [unowned self] result in
        switch result {
        case let .success(models):
            self.state = models.isEmpty ? .emptyContent : .content(models)
        case let .failure(error):
            self.state = .error(error)
        }
    }
}
```